build.gradle

```
@@ -17,10 +17,10 @@
 buildscript {
     ext {
         // Dependency versions.
-        activity_compose_version = '1.3.1'
-        compose_version = '1.1.0-beta01'
+        activity_compose_version = '1.4.0'
+        compose_version = '1.1.0-beta02'
         lifecycle_version = '2.4.0-rc01'
```

Nit: Could bump lifecycle as well
```suggestion
        lifecycle_version = '2.4.0'
```